Shopipy.AppointmentManagement/DTOs/AppointmentRequestDto.cs

```diff
@@ -0,0 +1,22 @@
+using System.ComponentModel.DataAnnotations;
```

Comment:
Why do you have appointment package if I see appointment logic in services already?